src/data/guilty_gear_xrd_rev_2/slayer.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Slayer
+moves:
+  normals:
+    - 5P
+    - 5K
+    - c.S
+    - f.S
+    - 5H
+    - 6P
+    - 6K
+    - '6[K]'
```

Comment:
Where is it shown that `6[K]` is a valid normal?

Reply:
I'm kind of irritated by this, actually. It's in the text for his 6K, not as a separate move.

`Hold down K to feint, 6[K]`

Reply:
gross

Reply:
i see it now.